### ISSUE TITLE
question

### ISSUE THREAD

**Issue by traderpedroso** [js-on/PyQDB#1]
how can I import date to Pandas Dataframe

res = qdb.exp(query="SELECT * FROM PyQDB")

import pandas as pd 

df = ?????

**Comment by simplylu** [comment]
Thanks for pointing that out. `qdb.exp` accidentally called the route `/exec` due to copy and paste. This caused, that the data returned wasn't CSV but JSON.
Fixed that in 4c34aaaac444b8fa7943f7219be60320fc7d9e2a and added the `output` parameter to `qdb.exp`, which is set to "csv" per default.

You can now do the following:
```py
>>> res = qdb.exp(query="SELECT * FROM PyQDB", output="pandas")
>>> type(res)
<class 'pandas.core.frame.DataFrame'>

# or
>>> res = qdb.exp(query="SELECT * FROM PyQDB", output="csv")
>>> type(res)
<class 'str'>
```

Please confirm that this solves your issue and I'll close that one.

**Comment by traderpedroso** [comment]
Phenomenal Thank you
